tests/performance/log-results.js

```diff
@@ -81,7 +81,7 @@ const data = new TextEncoder().encode(
 		branch,
 		hash,
 		baseHash,
-		timestamp: parseInt( timestamp, 10 ),
+		timestamp: timestamp,
```

Comment:
Nit: you could do:

```suggestion
		timestamp,
```

Reply:
But you probably want to keep the `parseInt` here, given that an environment variable that's passed to this string is always a string

Reply:
Check my reply in other comment